test/test_cli.go

```diff
@@ -418,7 +418,29 @@ func (s *CLISuite) TestRoute(t *c.C) {
 	t.Assert(r.Sticky, c.Equals, true)
 	t.Assert(r.Service, c.Equals, "foo")
 
-	// flynn route remove
+	// flynn route add domain path
+	pathRoute := app.flynn("route", "add", "http", route+"/path/")
+	t.Assert(pathRoute, Succeeds)
+	pathRouteID := strings.TrimSpace(pathRoute.Output)
+	assertRouteContains(pathRouteID, true)
+
+	// flynn route add domain path duplicate
+	dupRoute = app.flynn("route", "add", "http", route+"/path/")
+	t.Assert(dupRoute, c.Not(Succeeds))
+	t.Assert(dupRoute.Output, c.Equals, "conflict: Duplicate route\n")
+
+	// flynn route add domain path invalid
+	invRoute := app.flynn("route", "add", "http", route+"/path")
+	t.Assert(invRoute, c.Not(Succeeds))
+	t.Assert(invRoute.Output, c.Equals, "validation_error: Invalid route\n")
```

Comment:
Perhaps the error message could be nicer if it's just a missing trailling slash?